### DISCUSSION TITLE
Email images don't display when using custom port

### DISCUSSION THREAD

**Discussion by JOHLC** [dani-garcia/vaultwarden#1419] [Q&A] [answered]
### Subject of the issue
Email images don't display when using custom port

### Your environment

* Bitwarden_rs version: 1.19.0

* Install method: Docker
* Clients used: Any
* Reverse proxy and version: Nginx
* Version of mysql/postgresql: Unknown
![image](https://user-images.githubusercontent.com/63735432/108603578-c433db00-7376-11eb-8958-84c6980d2b4a.png)

* Other relevant information:

Possible workarounds: 
-- Use Base64 encoding so the image doesn't need to call a local resource
![image](https://user-images.githubusercontent.com/63735432/108603400-ad40b900-7375-11eb-896e-593c87939b02.png)
![image](https://user-images.githubusercontent.com/63735432/108603424-d19c9580-7375-11eb-946d-1963c052a9e8.png)
-- Allow user to specify port for static content like that
-- Serve the content from an online source instead of locally

### Steps to reproduce
Set custom https port number
Setup SMTP email 
Receive email from Bitwarden RS
Notice issue

### Expected behavior
Images should display in the emails from Bitwarden
![image](https://user-images.githubusercontent.com/63735432/108603614-f80f0080-7376-11eb-9fea-f46cc61c9af2.png)

### Actual behavior
Images broken in the email - in my case this is because I use a custom port and the email is hardcoded to default https port
Correct image url https://mydomain.com:5448/bwrs_static/logo-gray.png
Url used in the html email: https://mydomain.com/bwrs_static/logo-gray.png
![image](https://user-images.githubusercontent.com/63735432/108603118-132c4100-7374-11eb-8003-d4598826965e.png)

**Comment by JOHLC** [comment]
Neverminded I was able to resolve this by setting domain url with the port number in the admin panel
![image](https://user-images.githubusercontent.com/63735432/108603773-e712bf00-7377-11eb-88a0-4175c62370e5.png)

![image](https://user-images.githubusercontent.com/63735432/108603816-11fd1300-7378-11eb-9744-9c22971cfe92.png)

**Comment by BlackDex** [comment, marked as answer]
As the screenshot already tells you, your domain configuration is not correct. If you run bitwarden an other port, you also need to configure that as as `DOMAIN`, So include the `:5684`

**Comment by JOHLC** [comment]
@BlackDex  Yep, figured that out. Thank you!
